### PR TITLE
Updated to new Console commands system

### PR DESCRIPTION
**Pull request by Nogitsu**

### CODE REVIEW

File: Shared/Index.lua [hunk 1]
```diff
@@ -4,9 +4,6 @@ local function isLuaRun(sInput)
...
 end
 
-Console.Subscribe("PlayerSubmit", function(sText)
-    if (#sText >= (#sCmd + 2)) and isLuaRun(sText) then
-        load(string.sub(sText, (#sCmd + 2), #sText))()
-        return false
-    end
-end)
\ No newline at end of file
+Console.RegisterCommand( sCmd, function( ... ) -- Each word is an argument
```
Comment (Timmy-the-nobody) [inline Shared/Index.lua]: Could you please keep the previous syntax? (remove spaces, except between args)

File: Shared/Index.lua [hunk 2]
```diff
@@ -1,9 +1,3 @@
-local sCmd = "luarun"
-
-local function isLuaRun(sInput)
-    return sInput:sub(0, (#sCmd + 1)) == (sCmd.." ")
-end
-
-Console.RegisterCommand( sCmd, function( ... ) -- Each word is an argument
+Console.RegisterCommand( "luarun", function( ... ) -- Each word is an argument
```
Comment (Timmy-the-nobody) [inline Shared/Index.lua]: Could you please keep the previous syntax by removing extra spaces and just keeping the ones between args?

**Review by Timmy-the-nobody — CHANGES_REQUESTED** [review]
Check out the comment I left in the commit

File: Shared/Index.lua (hunk 2 above)
Comment (Nogitsu) [reply Shared/Index.lua]: I'll do it and then I'll kill you

**Pull request merged**